selfdrive/camerad/snapshot/snapshot.py

```diff
@@ -80,11 +101,13 @@ def snapshot():
 
   proc = subprocess.Popen(os.path.join(BASEDIR, "selfdrive/camerad/camerad"),
                           cwd=os.path.join(BASEDIR, "selfdrive/camerad"), env=env)
-  time.sleep(3.0)
+  time.sleep(4.0)
```

Comment:
remove the sleep

Reply:
It's overexposed for the first 2-4 seconds, and the focus code reports high focus during this time.

Reply:
handle this in get_snapshots